src/net.h

```diff
@@ -838,37 +838,89 @@ class CNode
         assert(false);
     }
 
+    /* These are the default connections that we use to connect with the
```

Comment:
Format these as doxygen comments (start with `/**`) https://www.doxygen.nl/manual/docblocks.html

Reply:
oof. so many options and yet I still missed the mark 😅 . moved the comments to the enum, but fixed this problem. thanks!